qiskit/finance/data_providers/wikipedia_data_provider.py

```diff
@@ -22,17 +22,19 @@
 import quandl
 from quandl.errors.quandl_error import NotFoundError
 
-from qiskit.finance.data_providers import (BaseDataProvider,
-                                           StockMarket, QiskitFinanceError)
+from ._base_data_provider import BaseDataProvider, StockMarket
+from ..exceptions import QiskitFinanceError
 
 logger = logging.getLogger(__name__)
 
 
 class WikipediaDataProvider(BaseDataProvider):
-    """Python implementation of a Wikipedia data provider.
+    """Wikipedia data provider.
+
     Please see:
     https://github.com/Qiskit/qiskit-tutorials/qiskit/finance/data_providers/time_series.ipynb
```

Comment:
```suggestion
    https://github.com/Qiskit/qiskit-tutorials/blob/stable/0.14.x/qiskit/advanced/aqua/finance/data_providers/time_series.ipynb
```

Reply:
Link is dead, let's change this we get the tutorial back up. Or remove entirely.